Xplat/src/main/java/vazkii/botania/common/item/relic/RingOfLokiItem.java

```diff
@@ -162,10 +162,10 @@ public static void breakOnAllCursors(Player player, ItemStack stack, BlockPos po
 		try {
 			for (BlockPos offset : cursors) {
 				BlockPos coords = pos.offset(offset);
-				BlockState state = player.getLevel().getBlockState(coords);
+				BlockState state = player.level().getBlockState(coords);
 				breaker.breakOtherBlock(player, stack, coords, pos, side);
-				ToolCommons.removeBlockWithDrops(player, stack, player.getLevel(), coords,
-						s -> s.is(state.getBlock()) && s.getMaterial() == state.getMaterial());
+				ToolCommons.removeBlockWithDrops(player, stack, player.level(), coords,
+						s -> s.is(state.getBlock()) /* && s.getMaterial() == state.getMaterial() */ ); // TODO
```

Comment:
I thought i'd done all of these already, but yeah my solution in most cases was to just remove the material check, not much to be done about it i think.

I feel it's also cleaner to just remove the code instead of commenting it out